Big-Spender/readme.md

```diff
@@ -85,15 +90,23 @@ INSERT YOUR QUERY HERE
 
 ```sql
 CREATE YOUR QUERY HERE
-```
+SELECT DATE_TRUNC('month', date) AS month, SUM(amount) AS total_amount
+FROM spends
+GROUP BY month
+ORDER BY month;
 
 **Farnoosh:** Thanks, that's really useful. We also need to know the total amount spent on each supplier. Can you help us with that?
 
 **You:** Sure thing. Here's the query for that:
 
 ```sql
-INSERT YOUR QUERY HERE
-```
+
+INSERT YOUR QUERY HERE```
+SELECT supplier_id, SUM(amount) AS total_amount
+FROM spends
+GROUP BY supplier_id;
```

Comment:
Yep, this looks goodl. could we use an alias for sum(amount) to help with keeping it cleaner by chance?